sdk/communication/communication-sms/samples/javascript/README.md

```diff
@@ -35,23 +37,25 @@ npm install
 
 2. Edit the file `sample.env`, adding the correct credentials to access the Azure service and run the samples. Then rename the file from `sample.env` to just `.env`. The sample programs will read this file automatically.
 
-3. Run the sample:
+3. Run whichever samples you like:
 
 ```bash
-node sendSms.js
+node sendMessage.js
```

Comment:
Did you mean to also rename the file `sendSms.js` to `sendMessage.js`? Right now, `sendMessage.js` doesn't exist.

Reply:
I did and then changed my mind. Clearly forgot to revert it. Good catch